<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <widget class="QListWidget" name="listDisplay">
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="descriptionDisplay"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionNew_List"/>
    <addaction name="actionOpen_List"/>
    <addaction name="actionClose_List"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuSort">
     <property name="title">
      <string>Sort</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionBy_Due_Date"/>
     <addaction name="actionAlpha_Numeric"/>
    </widget>
    <widget class="QMenu" name="menuClipboard">
     <property name="title">
      <string>Clipboard</string>
     </property>
     <addaction name="actionCut"/>
     <addaction name="actionCopy"/>
     <addaction name="actionPaste"/>
    </widget>
    <addaction name="actionMove_Up"/>
    <addaction name="actionMove_Down"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_Deselect_All"/>
    <addaction name="separator"/>
    <addaction name="menuSort"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove_all_checked_items"/>
    <addaction name="separator"/>
    <addaction name="menuClipboard"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_Deselect_All"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="actionEdit"/>
   <addaction name="separator"/>
   <addaction name="actionMove_Up"/>
   <addaction name="actionMove_Down"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_List">
   <property name="text">
    <string>New List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionOpen_List">
   <property name="text">
    <string>Open List</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add Item</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove Item</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit Item</string>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMove_Down">
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
  <action name="actionClose_List">
   <property name="text">
    <string>Close List</string>
   </property>
  </action>
  <action name="actionBy_Due_Date">
   <property name="text">
    <string>Due Date</string>
   </property>
  </action>
  <action name="actionAlpha_Numeric">
   <property name="text">
    <string>Alpha-Numeric</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionRemove_all_checked_items">
   <property name="text">
    <string>Remove all checked items</string>
   </property>
  </action>
  <action name="actionSelect_Deselect_All">
   <property name="text">
    <string>Select/Deselect All</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
